defaults/krill.conf

```diff
@@ -249,4 +249,36 @@
 # timing_child_certificate_valid_weeks = 52
 # timing_child_certificate_reissue_weeks_before = 4
 # timing_roa_valid_weeks = 52
-# timing_roa_reissue_weeks_before = 4
\ No newline at end of file
+# timing_roa_reissue_weeks_before = 4
+
+#
+#                 Suspend inactive child CAs
+#
+# NOTE: YOU DO NOT NEED THIS UNLESS YOU DELEGATE RESOURCES TO CHILD CAS
+#
+# If a child CA is deactivated, then it will no longer update its publication
+# point: i.e. its manifest(s) and CRL(s) will expire and it will not maintain
+# its ROAs. While this is a responsibility of the child CA itself, RPKI operators
+# may expect that RPKI parent CAs assume the responsibility to actively remove
+# such deactivated child CAs.
+#
+# The trouble is that there is no clear cut way for a parent to know that
+# a child CA had been deactivated. The best indication we have is by looking
+# at the last attempt by the child to send an RFC 6492 request to a parent. If
+# we haven't seen any attempt for longer than X hours we may be tempted to
+# suspend the child CA.
+#
+# By default Krill CAs will send RFC 6492 list every 10 minutes, and they will
+# republish manifests and CRLs 8 hours before they would expire. This means that
+# for Krill based child CAs we could use a value of 8 hours if we want to prevent
```

Comment:
I read "for Krill based child CAs" as meaning something should be done on the child CA system, but here I assume we are configuring the behaviour of the parent?